src/components/AlteriEditForm.vue

```diff
@@ -62,6 +62,50 @@
       </div>
     </div>
 
+    <div class="field is-horizontal" v-if="emoji">
+      <div class="field-label is-normal">
+        <label class="label">Emoji</label>
+      </div>
+      <br />
+      <div class="field-body">
+        <div class="field">
+          <div class="control">
+            <div class="selected-emoji">
+              {{
+                selectedEmoji.length < 1
+                  ? "Noch kein Emoji gewählt"
+                  : t("selectedEmoji") + selectedEmoji
+              }}
+            </div>
+            <button
+              v-if="selectedEmoji.length > 0"
+              @click="removeEmoji"
+              class="delete"
+              aria-label="remove emoji"
+            ></button>
```

Comment:
type="button" ergänzen , denn momentan wird der click auch aktiviert, wenn in einem Textfeld Enter getippt wird